tests/smoke.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+set -o pipefail
+
+TESTS="pwd, lsb_release -a, uname -a, ip a, lscpu, vmstat, lsblk, false"
```

Comment:
lsb_release isn't in my factory build?

also - the false test?

Reply:
false to show how the failure is handled. I can remove it. Same with lsb_release (which can be added to container as well)